### ISSUE TITLE
Web - версия

### ISSUE THREAD

**Issue by StepProgrammer** [alekseynemiro/MoneyBook#34]
Web - версия для тонких клиентов и компьютеров с исчерпанным запасом ёмкости жёсткого диска )))))

**Comment by alekseynemiro** [comment]
Да, так и было задумано. Но пока процесс разработки **Windows** версии немного подвис.

По задумке, для **Web** можно использовать **MoneyBook.Core**, как есть. Чтобы у каждого клиента была своя база **SQL Server Compact**. Чтобы клиенты могли базы скачивать и закачивать, ну и управлять :-)

Библиотека **MoneyBook.Core** уже должна быть готова для решения этой задачи.

**Comment by alekseynemiro** [comment]
@StepProgrammer, если есть интерес, могу тебя добавить в проект :-)

**Comment by StepProgrammer** [comment]
Давай попробуем. :-)

**Comment by alekseynemiro** [comment]
Добавил.

**Comment by StepProgrammer** [comment]
Спасибо. Кстати: постепенно наверное надо от десктопной версии отказываться..

**Comment by alekseynemiro** [comment]
> постепенно наверное надо от десктопной версии отказываться

Её бы для начала сделать :-)
